azure-quantum/azure/quantum/target/rigetti/target.py

```diff
@@ -47,6 +47,54 @@ class InputParams:
     This **must** be set true if using `Quil-T <https://pyquil-docs.rigetti.com/en/stable/quilt.html>`_.
     """
 
+    substitutions: Optional[Dict[str, List[List[float]]]] = None
+    """A dictionary of memory region names to the list of value vectors to write to that region.
+    
+    For example, a job with this Quil program:
+    
+    .. code-block::
+        
+        DECLARE ro BIT[2]
+        DECLARE theta REAL[2]
+        DECLARE beta REAL[1]
+        RX(theta[0]) 0
+        RX(theta[1]) 1
+        RX(beta) 2
+        MEASURE 0 ro[0]
+        MEASURE 1 ro[1]
+        MEASURE 2 ro[2]
+    
+    might be run with 
+    
+    .. highlight:: python
+    .. code-block::
+    
+        InputParams(
+            substitutions={
+                "theta": [
+                    [0.0, np.pi], [np.pi, 0.0]
```

Comment:
nit: just to keep it consistent with below.
```suggestion
                    [0.0, np.pi],
                    [np.pi, 0.0],
```